### DISCUSSION TITLE
CORS Error with X-Super-Client header

### DISCUSSION THREAD

**Discussion by RoastedPikachu** [helldivers-2/api#96] [Q&A] [answered]
Hello,

after updating API link in MajorOrderSection.tsx [c5dfe6d](https://github.com/RoastedPikachu/Helldivers_Help/commit/c5dfe6d13a640bd2fa39eb8c3d3a828bf52e5f91) i get CORS Error.

<details><summary>Logs</summary> 

Access to XMLHttpRequest at 'https://api.helldivers2.dev/raw/api/v2/Assignment/War/801' from origin 'https://helldivers-help.vercel.app' has been blocked by CORS policy: Request header field x-super-client is not allowed by Access-Control-Allow-Headers in preflight response.

</details>

I opened this discussion to ask if I'm doing something wrong or to ask you to add my fan site: [Helldivers.help](https://helldivers.help) and its vercel beta version: [vercel](https://helldivers-help.vercel.app) for Russian Helldivers 2 players to your CORS settings.

I can also assume, although I'm not sure, that the same error will occur due to the header X-Super-Contact.

Let me know what you think @dealloc .

**Comment by dealloc** [comment, marked as answer]
no mistake or error on your side, we forgot to update some configuration for browser clients
a fix is awaiting review in #97 sorry!

**Reply by RoastedPikachu** [reply]
Thanks for such a quick response, and also thanks for the last notification about the link API change

**Reply by dealloc** [reply]
@RoastedPikachu looks like this should be fixed, let me know if you run into any other problems

**Reply by RoastedPikachu** [reply]
Yes, it works great now, thanks
